src/web/Button.tsx

```diff
@@ -287,6 +287,13 @@ export class Button extends ButtonBase {
         if (this._longPressTimer) {
             Timers.clearTimeout(this._longPressTimer);
         }
+        if ('touches' in e) {
```

Comment:
I think this will cause a regression in button behaviour - touch down -> move -> touch up will trigger an on press, which isn't consistent with previous behaviour or react native

Reply:
Sorry for the reply my schedule does not allow me to handle this this week. Let me play with the test app to fully understand your concern. I will reply next week :)

Reply:
Got it!
I even caught the bug in https://github.com/dagatsoin/sproutch where a tab is triggered while the cursor is already out.

I just pushed the fix.

Reply:
I assume you've tried this with both mouse and touch?
The change looks reasonable but I haven't actually tried it locally :)

Reply:
Indeed, but do the QA plz, I don't won't to be responsible of the next Skype collapse :p